modules/web/js/ballerina/diagram2/views/default/sizing-util.js

```diff
@@ -1587,6 +1587,11 @@ class SizingUtil {
         const viewState = node.viewState;
         this.sizeStatement(node.getSource(), viewState);
         this.adjustToLambdaSize(node, viewState);
+
+        // Truncate the endpoint name to fit the statement box
+        if (TreeUtil.isEndpointTypeVariableDef(node)) {
+            viewState.endpointIdentifier = this.getTextWidth(node.getVariable().getName().value, 0, 100).text;
```

Comment:
Shall we get the value 100 to a meaningful variable and use it? Otherwise, it would be bit difficult to understand what the value meant to be.

Reply:
Fixed with 52c36d792ace3d77c1130ff45c3456ea660babc4